pages/micromaster/team.md

```diff
@@ -5,4 +5,5 @@
 |**Username**|**Join Date**|
 |------------|-------------|
 |[akhilmiryala](./profiles/akhilmiryala.md)|2020-11-05|
+|[Farzan Khoobchehr](./profiles/farzankh.md)|2020-12-30|
```

Comment:
Adding to team page is not part of this task.

Reply:
I deleted the row